packages/client/src/components/Card.tsx

```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
   return `${VALUES[value]} of ${SUIT[startCase(suit)]}`;
 }
 
-const suitList = Object.values(SUIT);
-function bgPosition([value, suit]: CardType) {
-  return `${(value - 1) * 11}% ${suitList.indexOf(suit) * 33}%`;
+function imgUrl([value, suit]: CardType) {
+  return require(`../images/${suit.toLowerCase()}/${padStart(`${value}`, 2, '0')}.jpg`).default;
 }
 
-const face = {
+const baseCard = {
   height: '13vw',
   width: '7.5vw',
   borderRadius: '0.75vw',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
   boxShadow: '1px 1px 5px rgba(0, 0, 0, 0.5)',
-  backgroundImage: `url(${bg})`,
-  backgroundSize: '1040%',
 };
 
-const Back = styled('div')`
-  height: 13.5vw;
-  width: 7.5vw;
-  border-radius: 0.75vw;
-  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.5);
-  background-color: blue;
-  border: 1px solid white;
-`;
-
+function cardStyle(card: CardType, faceDown?: boolean) {
+  return {
+    ...baseCard,
+    backgroundImage: `url(${faceDown ? back : imgUrl(card)})`,
+  };
+}
 interface CardProps {
-  className?: string;
   faceDown?: boolean;
```

Comment:
Should this just always exist?

Reply:
no it's fine to have it undefined, so you add it only when you need it. Will see in the future if we need something else

Reply:
Yeah, I just prefer more explicit definitions and defaulting the values (as you have done) rather that maybes.